portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase

```diff
@@ -398,14 +398,83 @@ definition {
 	}
 
 	@description = "Verify that an Allow Multiple Selections Property persists after being added to a Form via Fieldset"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test PreserveTheMultipleSelectionsProperty {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133473 PreserveTheMultipleSelectionsProperty pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		Form.gotoElementSetsTab();
```

Comment:
@fmarcio Can you please use `Navigator.gotoNavItem(navItem = "Element Sets");` instead to keep consistency? Thanks!

Reply:
> @fmarcio Can you please use `Navigator.gotoNavItem(navItem = "Element Sets");` instead to keep consistency? Thanks!

@carolmariaabb  Fixed! :-)